ci/job.yml

```diff
@@ -0,0 +1,161 @@
+- job:
+    name: centos-container-pipeline-service-container-index
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    properties:
+        - github:
+            url: https://github.com/CentOS/container-index
+    triggers:
+        - github-pull-request:
+            admin-list:
+              - kbsingh
+              - bamachrn
+              - mohammedzee1000
+              - dharmit
+              - navidshaikh
+            white-list:
+              - kbsingh
+              - bamachrn
+              - mohammedzee1000
+              - dharmit
+              - navidshaikh
+            cron: '* * * * *'
+            github-hooks: false
+            permit-all: true
+
+    wrappers:
+        - ansicolor
+    scm:
+        - git:
+            url: https://github.com/CentOS/container-index.git
+            skip-tag: True
+            git-tool: ci-git
+            branches: master
+            refspec: "+refs/pull/*:refs/remotes/origin/pr/*"
+            branches:
+                - "${ghprbactualcommit}"
+    builders:
+        - shell: |
+            git rebase origin/${ghprbTargetBranch}
+            curl https://raw.githubusercontent.com/CentOS/container-pipeline-service/master/ci/cccp_ci_container_index.sh > cccp_ci_container_index.sh
+            sh ./cccp_ci_container_index.sh
+
+- job:
+    name: 'centos-container-pipeline-service-ci-pr-openshift'
```

Comment:
We don't need quotes here. We didn't use quotes in other job names.

Reply:
Adding quotes is always better in jjb template name to avoid surprises. Added to all the jobs now.